src/main.rs

```diff
@@ -246,14 +274,30 @@ struct ReplayArgs {
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let opt = Cli::parse();
-    let filter = EnvFilter::from_default_env();
+
+    let log_level_filter = LevelFilter::from(opt.log);
+    CombinedLogger::init(vec![
+        TermLogger::new(
+            log_level_filter,
+            Config::default(),
+            TerminalMode::Mixed,
+            ColorChoice::Auto,
+        ),
+        WriteLogger::new(
+            log_level_filter,
+            Config::default(),
+            File::create("era_test_node.log").unwrap(),
+        ),
+    ])
+    .expect("failed instantiating logger");
 
     if opt.dev_use_local_contracts {
         if let Some(path) = env::var_os("ZKSYNC_HOME") {
             println!("+++++ Reading local contracts from {:?} +++++", path);
```

Comment:
We should update all `println!` from the repo in this PR if we are introducing the logger